src/util/MovieList.ts

```diff
@@ -0,0 +1,16 @@
+import { MovieElementData, MovieItem } from "../abstracts/type";
+
+const transformMovieItemsType = (
+  movieItems: MovieItem[]
+): MovieElementData[] => {
+  return movieItems.map(
+    ({ poster_path, id, title, vote_average }: MovieItem) => ({
```

Comment:
```suggestion
    ({ poster_path, id, title, vote_average }) => ({
```

이렇게하더라도 충분히 타입추론은 되는것 같군요 :)